bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -390,10 +397,12 @@ private int setSessionCookieAuthLogin() {
 
         try {
             ContentResponse response = httpClient.GET(url);
-            String pattern = "(?m)^\\s*\\r?\\n|\\r?\\n\\s*(?!.*\\r?\\n)";
-            String replacement = "";
-            logger.trace("HTTP Response ({}) Body:{}", response.getStatus(),
-                    response.getContentAsString().replaceAll(pattern, replacement));
+            if (logger.isTraceEnabled()) {
+                String pattern = "(?m)^\\s*\\r?\\n|\\r?\\n\\s*(?!.*\\r?\\n)";
+                String replacement = "";
+                logger.trace("HTTP Response ({}) Body:{}", response.getStatus(),
+                        response.getContentAsString().replaceAll(pattern, replacement));
+            }
```

Comment:
This code seems to be repeated a lot, can you refactor it into a separate method and call it instead?

Reply:
Fixed!